src/ensembl/src/content/app/help/components/help-menu/HelpMenu.tsx

```diff
@@ -114,25 +127,29 @@ const Submenu = (props: SubmenuProps) => {
   const [childItems, setChildItems] = useState<MenuItem[] | null>(null);
   const dispatch = useDispatch();
 
+  const { trackArticleLinkClicked } = useHelpAppAnalytics();
+
   useEffect(() => {
     setChildItems(null);
   }, [props.items]);
 
-  const onLinkClick = (url: string) => {
+  const onLinkClick = (item: MenuArticleItem) => {
     // hopefully, the url is an internal one;
     // might need extra logic if we can have external urls in the menu
     props.onLinkClick();
-    dispatch(push(url));
+    dispatch(push(item.url));
+    trackArticleLinkClicked(item.name, item.type);
   };
 
   const renderedMenuItems = props.items.map((item, index) => {
     const className = classNames(styles.submenuItem);
     const props: Record<string, unknown> = {};
```

Comment:
I know this was not added in this PR. However naming this variable as `props` in a React component file is misleading. Can we name this something else like `eventObj`?

Reply:
But this variable here does represent the collections of props — the ones that are passed to the child.